deepchecks/vision/vision_data.py

```diff
@@ -224,6 +225,25 @@ def batch_to_images(self, batch) -> Sequence[np.ndarray]:
         """
         raise DeepchecksNotImplementedError('batch_to_images() must be implemented in a subclass')
 
+    def calc_properties(self, raw_data: List = None, properties_list: List[Dict] = None) -> Dict[str, list]:
```

Comment:
Here lies my biggest concern - how do override this method in a monitoring context to retrieve the appropriate properties? Because this method receives the raw data, it has no idea what is the index of the image, and so no way to query the properties for the right image from the DB...
@TheSolY @ItayGabbay Am I right about this?

Reply:
@noamzbr raw data can be any kind of list, not necessarily images, so it can also be a list of precalculated results with a single property that is the identity function.  As to managing indices in the batch_wrapper- I assumed we can implement it similar to _do_static_pred, am I wrong here?